package.json

```diff
@@ -124,10 +125,45 @@
         "contents": "No open pull request found.\n[Try again](command:codacy.pr.load)",
         "when": "codacy:hasProject && codacy:isGitRepository && Codacy:RepositoryManagerStateContext == Loaded && Codacy:PullRequestStateContext == NoPullRequest"
       },
+      {
+        "view": "codacy:mcp",
+        "contents": "Control your AI generated code with [Codacy Guardrails](https://www.codacy.com/guardrails/).\n[Install Guardrails](command:codacy.configureGuardrails)\n\nGets everything set up by installing Codacy CLI, setting up Codacy MCP Server, and adding the necessary agent rules files.\n\nYou can also choose to [only install the MCP Server](command:codacy.configureMCP)",
+        "when": "codacy:supportsMCP && !codacy:mcpConfigured && !codacy:cliInstalled && codacy:canInstallCLI && !codacy:cliInstalling"
+      },
+      {
+        "view": "codacy:mcp",
+        "contents": "Set up Codacy MCP Server to get access to Codacy information from your agent.\n[Install MCP Server](command:codacy.configureMCP)\n\nTo control your AI generated code with Codacy Guardrails, you need to be able to install the Codacy CLI (macOS, Linus, or Windows via WSL). [Learn more about Codacy Guardrails](https://www.codacy.com/guardrails/)",
+        "when": "codacy:supportsMCP && !codacy:mcpConfigured && !codacy:canInstallCLI"
+      },
+      {
+        "view": "codacy:mcp",
```

Comment:
in case everything is installed we lost this message Felipe planned "MCP Server is installed in the IDE settings and a file with rules for the agent in your project’s .codacy directory was created." I think it worth to add it

Reply:
Also the Learn more disappeared